### PR TITLE
[ADD][9.0] [purchase_product_menu]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This modules adds the Product Variants to the Purchases menu.
# Usage
# . Go to 'Purchases / Configuration / Settings' and set the Product Variants

   option 'Products can have several attributes, defining variants (Example:
   size, color,...)'

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @pedrobaeza

File: purchase_product_menu/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Copyright 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Products Menu",
+    "version": "9.0.1.0.0",
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
```
Comment (pedrobaeza) [inline purchase_product_menu/__openerp__.py]: Please put Tecnativa instead.

**Comment by pedrobaeza** [comment]
:+1: except minor comment

**Comment by pedrobaeza** [comment]
Thanks! Again :+1:

**Comment by JordiBForgeFlow** [comment]
cc @eficent.

**Review by AaronHForgeFlow — COMMENTED** [review]
The .po files will not be used anymore, right?

**Review by AaronHForgeFlow — COMMENTED** [review]
Tested in local (runbot is down) + code review :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
code review :+1:

**Pull request closed without merging**